<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpticalFlowSelectorDialog</class>
 <widget class="QDialog" name="OpticalFlowSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpticalFlowSelectorDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>221</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioLukasKanade">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLukasKanade">
        <property name="text">
         <string>LukasKanade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configLukasKanade">
        <property name="text">
         <string>config</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioFarneback">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFarneback">
        <property name="text">
         <string>Farneback</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configFarneback">
        <property name="text">
         <string>config</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioBlockMatching">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBlockMatching">
        <property name="text">
         <string>Blockmatching</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configBlockMatching">
        <property name="text">
         <string>config</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QRadioButton" name="radioBrox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelBrox">
        <property name="text">
         <string>Brox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configBrox">
        <property name="text">
         <string>config</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="radioDualTVL1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDualTVL1">
        <property name="text">
         <string>DualTVL1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configDualTVL1">
        <property name="text">
         <string>config</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="radioHS">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelHS">
        <property name="text">
         <string>Horn/Schunk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="configHS">
        <property name="text">
         <string>config</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
